src/core/ConfigManager.cpp

```diff
@@ -123,6 +123,7 @@ void ConfigManager::upgrade_1_1_91()
 }
 
 
+
```

Comment:
Since we're not touching ConfigManager anymore, let's make this not touch the file either.

Reply:
No problem, I didn't realize I added a blank line until I pushed the changes. On line 146 though I actually deleted a blank space that was probably left in the empty line. Should I leave it there too?

Reply:
I left the Tab where it was to remove the file from the "Files changed" list. I believe it will probably be removed later when the code base is refactored through clang.